schemas/blocks/definitions.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "content": {
+    "description": "Block level content block",
+    "type": "object",
+    "properties": {
+      "title": {
+        "type": "string",
+        "description": "The top-level heading on the page"
+      },
+      "contents": {
+        "$ref": "../common_definitions.json#/contents"
+      }
+    }
```

Comment:
Same here, worth adding `"additionalProperties": false,`
We should also enforce the `required` keys to prevent invalid schemas.